QUARKUS-2453.md

```diff
@@ -0,0 +1,15 @@
+# QUARKUS-2453 - Deprecate quarkus-reactive-routes extension
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-2453
+
+Deprecate `quarkus-reactive-routes` extension starting with RHBQ 2.13, for RHBQ 2.7 the extension is on supported level.
+
+## Scope of the testing
+
+Ensure `deprecated` label is rendered for `quarkus-reactive-routes` extension on https://code.quarkus.redhat.com/ pre-stage instance.
```

Comment:
According to the `QUARKUS-2527` there are two `deprecated` labels nowadays. I presume, we must ensure, that both are rendered? In such a case, that should be mentioned explicitly.

Reply:
`quarkus-reactive-routes` is deprecated just for product, not for upstream. So just one label is expected. Thus the line seems relevant to me.

Reply:
```suggestion
Ensure `deprecated` label is rendered in `redhat-support`  field for `quarkus-reactive-routes` extension on https://code.quarkus.redhat.com/ pre-stage instance.
```